lib/presentation/widget/upload_request_inside/upload_request_inside_widget.dart

```diff
@@ -520,4 +519,14 @@ abstract class UploadRequestInsideWidgetState extends State<UploadRequestInsideW
         return buildRecipientMultipleSelectionBottomBar(context, state.getAllSelectedRecipient());
     }
   }
+
+  Widget editUploadRequestRecipientAction(BuildContext context, UploadRequest entry) {
```

Comment:
`Edit` only for `UploadRequest` in `status == Active`, am I right?

Reply:
Also pending status.